lib/containers/k8s.pm

```diff
@@ -72,6 +72,11 @@ sub install_k3s {
     script_retry("kubectl get serviceaccount default -o name", delay => 60, retry => 3);
     # Await k3s to be ready and the api is accessible
     script_retry("kubectl get pods --all-namespaces", delay => 60, retry => 3);
+    assert_script_run("systemctl is-active k3s");
```

Comment:
If k3s is not active, the previous command can not succeed, so this one is not necessary IMHO.

Reply:
I suspect that sometimes the k3s fall down, so I would like to keep this or move to later. what do you think?

Reply:
Ok, removed. We have the sytemctl status k3s in case of failure